apps/web/.env.example

```
@@ -23,4 +23,12 @@ VITE_PUBLIC_VERCEL_BLOB_BASE_URL=
 VERCEL_RENDERER_PROJECT=
 VERCEL_TOKEN=
 INNGEST_SIGNING_KEY=
-INNGEST_EVENT_KEY=
\ No newline at end of file
+INNGEST_EVENT_KEY=
+
+# New for docs as code, later to add in docker compose and sst
+GITHUB_APP_ID=
+GITHUB_APP_CLIENT_ID=
+GITHUB_APP_CLIENT_SECRET=
+GITHUB_APP_PRIVATE_KEY= # one-line PEM with literal \n (e.g. "-----BEGIN...\\n...\\n-----END RSA PRIVATE KEY-----\\n")
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)